docs/ja/user-guide/configuration/index.md

```diff
@@ -15,80 +15,98 @@ toc:
 
 各プロパティ名にマウスカーソルを乗せるとそれらの説明が表示されます。
 
-
 <div class="yaml-docs">
-
+</div>
+<div data-md-type="block_html">
 <pre class="example">
-<a href="#workflow"><span class="key">workflow</span>:
-    - <span class="value">publish</span>
-    - <span class="key">parallel</span>:
-        - <span class="key">series</span>:
-            - <span class="value">deploy-east</span>
-            - <span class="value">validate-east</span>
-        - <span class="key">series</span>:
-            - <span class="value">deploy-west</span>
-            - <span class="value">validate-west</span></a>
 <a href="#shared"><span class="key">shared</span>:</a>
     <a href="#environment"><span class="key">environment</span>:
     <span class="key">NODE_ENV</span>: <span class="value">test</span></a>
     <a href="#settings"><span class="key">settings</span>:</a>
         <a href="#email"><span class="key">email</span>:
     <span class="key">addresses</span>: <span class="value">[test@email.com, test2@email.com]</span>
     <span class="key">statuses</span>: <span class="value">[SUCCESS, FAILURE]</span></a>
+    <a href="#annotations"><span class="key">annotations</span>:
+    <span class="key">beta.screwdriver.cd/my-annotation</span>: <span class="value">my-data</span></a>
```

Comment:
The original example has more annotations examples.
see: https://github.com/screwdriver-cd/guide/commit/287a7e4078d7e3c8549e6e5fac1c7eee47f6c1f0

Reply:
added. thanks!